src/osd/OSD.cc

```diff
@@ -6525,7 +6519,15 @@ void OSD::_dispatch(Message *m)
 
     // -- need OSDMap --
 
-  default:
+  case MSG_OSD_PG_CREATE:
+  case MSG_OSD_PG_NOTIFY:
+  case MSG_OSD_PG_QUERY:
+  case MSG_OSD_PG_LOG:
+  case MSG_OSD_PG_REMOVE:
+  case MSG_OSD_PG_INFO:
+  case MSG_OSD_PG_TRIM:
+  case MSG_OSD_BACKFILL_RESERVE:
+  case MSG_OSD_RECOVERY_RESERVE:
```

Comment:
No, an unrecognized message asserts in dispatch_op later; it just seemed better to enumerate them here (and not assert on a stray message type) than to assert in dispatch_op.  (This after i spent several minutes trying to figure out which messages took this path.)